spec/fixtures/models/basic_dir_models.rb

```diff
@@ -25,7 +25,7 @@ def image
 class ChildImportDirModel
   include DirModel::Model
   include DirModel::Import
-  file :metadata, regex: ->(foreign_value) { "Zones\\/(?<sector_name>#{foreign_value})\\/Zone_(?<zone_id>.*)\\.(?<extension>json)" }
+  file :metadata, regex: ->(foreign_value) { /Zones\/(?<sector_name>#{foreign_value})\/Zone_(?<zone_id>.*)\.(?<extension>json)/i }
```

Comment:
I use String on `foreign_value` because of that 

```
>> Regexp.new("Foo/#{Regexp.quote('R+4 Logement 844 T2')}")
=> /Foo\/R\+4\ Logement\ 844\ T2/
```
vs
```
>> Regexp.new("Foo/R+4 Logement 844 T2")
=> /Foo\/R+4 Logement 844 T2/
```